vhdbuilder/packer/install-dependencies.sh

```diff
@@ -112,10 +112,15 @@ if [[ $OS == $MARINER_OS_NAME ]]; then
     disableSystemdIptables
     forceEnableIpForward
     setMarinerNetworkdConfig
-    enableDNFAutomatic
     fixCBLMarinerPermissions
     addMarinerNvidiaRepo
     overrideNetworkConfig || exit 1
+    if grep -q "kata" <<< "$FEATURE_FLAGS"; then
+      enableMarinerKata
+    else
+      # Leave automatic package update disabled for the kata image
+      enableDNFAutomatic
```

Comment:
cc @paulgmiller

this fancy kata containers image won't have dnf automatic updates enabled for compat reasons. 

(no action for mariner folks)